actions/hls_image_filter/hw/action_pixel_filter.H

```diff
@@ -0,0 +1,49 @@
+#ifndef __ACTION_PIXEL_FILTER_H__
+#define __ACTION_PIXEL_FILTER_H__
+
+/*
+ * Copyright 2020 International Business Machines
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <ap_int.h>
+
+#include "hls_snap_1024.H"
+#include <action_pixel_filtering.h> /* action struture definition */
+
+#define PATTERN_SIZE BPERDW_1024
+#define CARD_DRAM_SIZE (1 * 1024 *1024 * 1024)  //Maximum size in bytes (depends on the card used)
+#define MAX_NB_OF_BYTES_READ  (4 * 1024)
+#define MAX_NB_OF_WORDS_READ MAX_NB_OF_BYTES_READ/BPERDW_1024
+
+typedef char word_t[BPERDW_1024];
+typedef snapu64_t address_t;
+
+#define PATTERN_SIZE BPERDW_1024
+#define TEXT_SIZE    4096 * MAX_NB_OF_BYTES_READ // used for streaming only
+
+typedef char word_t[BPERDW_1024];
+//---------------------------------------------------------------------
+// This is generic. Just adapt names for a new action
+// CONTROL is defined and handled by SNAP
```

Comment:
CONTROL is defined and handled by SNAP => OC_ACCEL

Reply:
ok